modules/02_welfare/utilitarian/declarations.gms

```diff
@@ -21,6 +21,8 @@ $endif.inconv
 $IFTHEN.INCONV_bioSwitch "%cm_INCONV_PENALTY_FESwitch%" == "on"
 p02_inconvPen_Switch_Track(ttot,all_regi)                       "Parameter to track magnitude of inconvenience penalty for bio/synfuel share switching [share of consumption]"
 $ENDIF.INCONV_bioSwitch
+
+p02_inconvPen_NonEn_Switch_Track(ttot,all_regi)                  "Parameter to track magnitude of inconvenience penalty for bio/synfuel non-energy fuel switching [share of consumption]"
```

Comment:
and what about policy scenarios where, e.g., available bioenergy could go into feedstocks? 
@fschreyer

Reply:
This is to avoid switching between synfuels and biofuels in non-energy use. They both have the same emissions factors and it is no difference to the model whether to put them into energy or non-energy use. 

In the beginning we used to have switching behavior, remember? I guess we still have the plots somewhere.

Reply:
If there is a CO2 price, there is a direct difference between mitigation costs due to the different emission factors that would always favor using synfuels and biomass prioritarily in energy before non-energy demand by the model. In the baseline case, the equation `q37_FossilFeedstock_Base` will enforce the use of only fossils. I wouldn't see the case then that "it is no difference to the model whether to put them into energy or non-energy use".

About the switch between synfuels and biomass use inside nonenergy, I would see very difficult that both share the exact same costs in any scenario. I would see even more improbably that these exact same costs would be kept constant over time.

I remember we had something like that, but probably equation `q37_FossilFeedstock_Base` was added afterwards, and we still did not have a proper formulation working.

Reply:
This is what happens when we remove the penalties for switching between bio and synfuels, so I think we have to keep them:
![image](https://user-images.githubusercontent.com/89921124/183087237-984dd604-682d-463d-af2e-8ee76b6ce8b7.png)

Reply:
These charts show me another issue. 
You are decarbonizing two thirds of non-energy liquids by 2030 already, using huge amounts of hydrogen and biomass for that. 
Is this even feasible? Maybe something else wrong in the formulation.
Did you check if the emission factors for the non-energy part are not too big? Or if they are being applied correctly in the emissions equation?
I would expect non-energy mitigation to not be competitive at all as a decarbonization strategy until much latter if you consider the simple "small emission factor" formulation done so far.

Reply:
I removed these penalties as they are not needed in the new formulation